.github/workflows/validate-format.yml

```diff
@@ -0,0 +1,14 @@
+name: Format
+on:
+  pull_request:
+    paths:
+      - "**/*.swift"
+      - ".github/workflows/validate-format.yml"
+
+jobs:
+  format:
+    name: Validate Swift formatting
+    runs-on: macos-14
```

Comment:
```suggestion
    runs-on: macos-12
```

Reply:
(thought you wanted 14 since you used `latest`)

Reply:
14 is not available. latest == 12

Reply:
Nope: https://github.com/actions/runner-images/tree/main?tab=readme-ov-file#available-images